experimental/InjectiveResolutions/src/InjectiveResolutions.jl

```diff
@@ -1064,5 +1064,4 @@ export zeroth_local_cohomology
 
 export MonoidAlgebra
 export MonoidAlgebraIdeal
-export MonoidAlgebraElem
-
+export MonoidAlgebraElem
```

Comment:
```suggestion
export MonoidAlgebraElem

```
please keep the trailing newline character